recipe/meta.yaml

```diff
@@ -69,7 +69,7 @@ requirements:
     - gunicorn <21  # [not win]
     - waitress <3  # [win]
     - scikit-learn <2.0
-    - pyarrow <12,>=4.0.0
+    - pyarrow <13,>=4.0.0
```

Comment:
Seems that upstream they are inconsistient with what they prefer the max version to be. Pyarrow maximum version is said to be 12 instead of 13 [here](https://github.com/mlflow/mlflow/blob/9ee9f4ebd9172c2a17af20ec1d9d506843057d13/requirements/core-requirements.yaml#L59C24-L59C24)
versus in [core-requirements.txt](https://github.com/mlflow/mlflow/blob/9ee9f4ebd9172c2a17af20ec1d9d506843057d13/requirements/core-requirements.txt#L16). I think its safe to assume it is 13. LGTM :)

Reply:
Since the yaml file is not read by setup.py, the txt file is what matters